.github/workflows/ms.network.virtualnetworks.virtualnetworkpeerings.yml

```diff
@@ -21,12 +21,12 @@ on:
     paths:
       - '.github/actions/templates/**'
       - '.github/workflows/ms.network.virtualnetworks.virtualnetworkpeerings.yml'
```

Comment:
- Please remove this file. We don't want pipelines for child resources
- Also remove the `.parameters` folder from the child resource. The contained test-parameters for the child should move to a top-level parameter file

Reply:
done